Analysis/src/TypeVar.cpp

```diff
@@ -1072,14 +1072,16 @@ static std::vector<TypeId> parseFormatString(TypeChecker& typechecker, const cha
                 continue;
 
             // we just ignore all characters (including flags/precision) up until first alphabetic character
-            while (i < size && !(data[i] > 0 && isalpha(data[i])))
+            while (i < size && !(data[i] > 0 && (isalpha(data[i])) || data[i] == '*'))
```

Comment:
I could have a separate check like the `%` one, but I wanted it to be painless to add flags to %*